### ISSUE TITLE
v0.0.0.65 Unknown inv type "mn winner ..."

### ISSUE THREAD

**Issue by walkjivefly** [Crowndev/crown#164]
Seen in collateral wallet debug.log
```
2022-10-30T17:32:19Z received: inv (109 bytes) peer=31
2022-10-30T17:32:19Z got inv: mn winner 36882c69cc47ac94ca754cc69861258702fd377b195aa73936848657b3ca2a5b  new peer=31
2022-10-30T17:32:19Z askfor mn winner 36882c69cc47ac94ca754cc69861258702fd377b195aa73936848657b3ca2a5b  0 (00:00:00) peer=31
2022-10-30T17:32:19Z Unknown inv type "mn winner 36882c69cc47ac94ca754cc69861258702fd377b195aa73936848657b3ca2a5b" received from peer=31
2
```

**Comment by BlockMechanic** [comment]
Just a warning, not really an issue. Solved in 0.0.0.66

**Comment by walkjivefly** [comment]
Confirmed appears resolved in 0.0.0.68
